.github/workflows/test.yml

```diff
@@ -26,4 +26,11 @@ jobs:
       - name: Test LiveSPICEVst build
         shell: pwsh
         working-directory: LiveSPICEVst
-        run: dotnet publish -c Release --framework net6.0-windows /p:DebugType=None /p:UseSharedCompilation=false  /p:UseRazorBuildServer=false
\ No newline at end of file
+        run: dotnet publish -c Release --framework net6.0-windows /p:DebugType=None /p:UseSharedCompilation=false  /p:UseRazorBuildServer=false
+      - name: Run circuit tests
+        shell: pwsh
+        working-directory: Tests
+        run: |
+          dotnet publish -c Release --framework net6.0-windows /p:DebugType=None /p:UseSharedCompilation=false  /p:UseRazorBuildServer=false \
+          bin/Release/net6.0-windows/CircuitTests.exe test "Circuits\*.schx" \
```

Comment:
Tried this locally, and in my case the path is `bin\Release\net6.0-windows\publish`

Reply:
Also, you don't have to publish if you just want to run it. This should be enough:
`dotnet run -c Release --framework net6.0-windows test "Examples\*.schx"`